charts/opentelemetry-operator/Chart.yaml

```diff
@@ -12,4 +12,4 @@ maintainers:
   - name: jaronoff97
   - name: TylerHelmuth
 icon: https://raw.githubusercontent.com/cncf/artwork/a718fa97fffec1b9fd14147682e9e3ac0c8817cb/projects/opentelemetry/icon/color/opentelemetry-icon-color.png
-appVersion: 0.117.0
+appVersion: 0.119.0
```

Comment:
Should we do `118` first?

Reply:
Ya need to merge https://github.com/open-telemetry/opentelemetry-helm-charts/pull/1536 first

Reply:
@frzifus we're ready to go to v0.119.0

Reply:
Sorry,  I ve been OOO - I will  update the PR in a bit.